task.aspx

```diff
@@ -115,14 +122,25 @@
                         </center>
                      </div>
                   </div>
+
                   <div class="row">
+                      <asp:SqlDataSource ID="SqlDataSource1" runat="server" ConnectionString="<%$ ConnectionStrings:LogBookConnectionString %>" SelectCommand="SELECT * FROM [task_detail_tbl]"></asp:SqlDataSource>
```

Comment:
### Doable, but bad practice. 

While this is a given feature and is doable, you should try to minimize the usage of this. 

Mainly because we should **NOT** try to mix backend with frontend code. If you use Blazor or Razor or MVC, this is doable because it was meant to be that way. 

Even with React.js, it is meant to be done that way.

But with ASP.NET WebForms, we try not to do it too much. Mainly because ASP.NET is already messy as it is.